lsst-epo/rubin-obs-client
# feat: triad logo updates

Updates to the header to use the new triad logos, with updated layouts to accommodate. Also includes:

- Refactor of header component structure
- Removal of user profile page (never been set up)
- Replace navbar hider with `react-headroom`
- Move auth modals out of header component
- Get logos from Canto assets via Craft 
- Use art direction for desktop vs. mobile logo
- Next.js image caching

To test: 

- Using the matching API branch, set either the desktop or desktop + mobile logos from Craft. The triad logos are not there yet, so the existing logos are fine to use. 

Desktop (>1500px, may depend on pixel density i.e. my Macbook uses the mobile display)
- on scroll down, header should slide away once window is past the bottom of the header
- on scroll up, header should appear 
- top level
  - should show the larger logo set
  - should show the search bar
  - should show the language toggle
    - both components should have no change in function
- lower level
  - should show navigation items on click
  - when navigation items are open, header should be pinned on scroll down
  - on click outside navigation items, list should close 
  - on click another navigation item, previous list should close and new list should open
Mobile
- top level
  - should show a mobile-targeted logo if set
  - should show default logo if no mobile-targeted logo is set
- lower level
  - should show language toggle
  - should show search bar
  - should show hamburger icon button
  - on hamburger click
    - header should stay pinned
    - menu should slide out
    - on click nav item
      - subnav should slide out
      - header should stay pinned
    - on click outside subnav
      - subnav should close
      - header should be unpinned
